db/migrations/dedupe_plugin_repo_owner.py

```diff
@@ -0,0 +1,87 @@
+"""Remove case-sensitive duplicates in the plugins table"""
+
+import rethinkdb as r
+
+import db.plugins
+import db.util
+
+r_conn = db.util.r_conn
+
+LOG_FILE = 'deleted_slugs.log'
+
+
+def dupe_log_line(canonical, dupes):
+    return '%s: %s\n' % (canonical, ', '.join(dupes))
+
+
+def merge_plugins(plugins):
+    def reducer(new, old):
+
+        # Use the plugin with the shortest slug as the new plugin
+        if len(old['slug']) < len(new['slug']):
+            new, old = (old, new)
+
+        new = db.plugins.update_plugin(old, new)
+
+        # Preserve categories
+        if new['category'] == 'uncategorized':
+            new['category'] = old['category']
+
+        # Merge tags
+        new['tags'] = list(set(new['tags'] + old['tags']))
```

Comment:
I was thinking sorted, that's what I get for being tired and code reviewing at the same time. Thanks for the correction.